### ISSUE TITLE
Allow scheduling of more than 10k jobs on CC

### ISSUE THREAD

**Issue by dantp-ai** [yiwan-rl/AlphaEx#2]
One approach is to have an extra parameter `offset` for the original index:

`my index = (OFFSET-1)*10000 + job array index`

**Comment by yiwan-rl** [comment]
Agree, the offset should be passed into sbatch script by submitter through --export. And the user can then use 

`my index = (OFFSET-1)*10000 + job array index`

in the sbatch script.
